src/dialects/yugabyte/connection-manager.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const _ = require('lodash');
+const dataTypes = require('../../data-types');
+const PostgresConnectionManager = require('../postgres/connection-manager');
+
+class ConnectionManager extends PostgresConnectionManager{
+  constructor(dialect, sequelize) {
+    sequelize.config.port = sequelize.config.port || 5433;
+    super(dialect, sequelize);
+    this._clearDynamicOIDs();
+    this._clearTypeParser();
+    this.refreshTypeParser(dataTypes.yugabyte);
+  }
+
+}
+
+module.exports = ConnectionManager;
+module.exports.ConnectionManager = ConnectionManager;
+module.exports.default = ConnectionManager;
```

Comment:
whats the difference/need for line 18,19,20

Reply:
These are the different exports methods for the module but I guess we only require 18th line for exporting but I saw this kind of exports in different ways in postgres and other dialects as well and not only for `connection-manager` , it is used in other modules as well.

Reply:
ok